build.gradle.kts

```diff
@@ -32,6 +32,8 @@ dependencyAnalysis {
     all {
       onAny {
         severity("fail")
+        // Weird issue due to wire.
+        exclude("() -> java.io.File?")
```

Comment:
I don't even comprehend this. How could the plugin possibly thing that a lambda is a dependency?

Reply:
might be due to https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/issues/884